src/ontology/edits/oeo-model.omn

```diff
@@ -3680,6 +3743,61 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1791",
         OEO_00360020
     
     
+
+Class: OEO_00400012
+
+    Annotations: 
+        OEO_00020426 "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1829
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1945",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Uncertainty of a model is a qualtity of a model that represents any departure from the unachievable ideal of complete determinism.",
+        <http://purl.obolibrary.org/obo/IAO_0000119> "https://www.researchgate.net/publication/264816512_Classifying_and_communicating_uncertainties_in_model-based_policy_analysis",
+        rdfs:comment "Uncertainty of a model is a situation of inadequate information, which can be of three sorts: inexactness, unreliability, and border with ignorance.",
+        rdfs:label "uncertainty of a model"@en
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/BFO_0000019>,
+        OEO_00400037 some OEO_00400013,
+        OEO_00400038 some OEO_00400017
+    
+    
+Class: OEO_00400013
+
+    Annotations: 
+        OEO_00020426 "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1829
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1945",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Nature of uncertainty is a quality of a model that represents the core reason for uncertainty within it.",
+        <http://purl.obolibrary.org/obo/IAO_0000119> "https://www.researchgate.net/publication/264816512_Classifying_and_communicating_uncertainties_in_model-based_policy_analysis",
+        rdfs:label "nature of uncertainty"@en
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/BFO_0000019>
+    
+    
+Class: OEO_00400017
+
+    Annotations: 
+        OEO_00020426 "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1829
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1945",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Level of uncertainty is a quality of a model that represents the degree of an uncertainty.",
+        <http://purl.obolibrary.org/obo/IAO_0000119> "https://www.researchgate.net/publication/264816512_Classifying_and_communicating_uncertainties_in_model-based_policy_analysis",
+        rdfs:label "level of uncertainty"@en
+    
+    SubClassOf: 
+        <http://purl.obolibrary.org/obo/BFO_0000019>
+    
+    
+Class: OEO_00400022
+
+    Annotations: 
+        OEO_00020426 "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1829
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1945",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Location of uncertainty is a role of an oeo class that has the disposition to be the part of a system in which an uncertainty occurs.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "Location of uncertainty role is a role of an entity in which a model uncertainty is located in.",
```
We could also introduce a disposition, respectively, but I am not sure how useful that is... I'd propose to simplify the wording.